crates/dbsp/src/circuit/circuit_builder.rs

```diff
@@ -3258,6 +3815,33 @@ where
             (node, output_stream)
         })
     }
+
+    fn add_replay_edges(&self, stream_id: StreamId, replay_stream: &dyn StreamMetadata) {
```

Comment:
I feel I could use some documentation here.

Reply:
It's documented in the trait declaration (you commented about that too).